modules/installation-configuration-parameters.adoc

```diff
@@ -202,11 +202,11 @@ You can customize your installation configuration based on the requirements of y
 // https://bugzilla.redhat.com/show_bug.cgi?id=2020416
 // Once BM UPI supports dual-stack, uncomment all the following conditionals and blocks
 
-ifndef::bare,vsphere[]
+ifndef::vsphere[]
 Only IPv4 addresses are supported.
 endif::[]
```

Comment:
```suggestion
endif::bare,ibm-power,ibm-z,vsphere[]
```
I would close like this. Otherwise, not sure if this might break something on the portal.

Reply:
Updated.